ra/registration-authority.go

```diff
@@ -32,6 +32,11 @@ import (
 // TODO(jsha): Read from a config file.
 const DefaultAuthorizationLifetime = 300 * 24 * time.Hour
 
+// 36 hour default pending authorization lifetime.  If you can't respond to a
+// challenge this quickly, then you need to request a new challenge.
+// TODO(rlb): Read from a config file
+const DefaultPendingAuthorizationLifetime = 36 * time.Hour
```

Comment:
One of the large interested organizations requested we keep this >= 1 week, as it gives time for their customers to reconfigure legacy equipment after, to paraphase, 'starting the ACME wizard.' 

I'd recommend taking this out to 1 week; it's much better than 300 days!